README.md

```diff
@@ -1,36 +1,41 @@
-# Yo Azure Data Studio - Extension and Customization Generator
```

Comment:
Do we want to name this just the 'Azure Data Studio Extension Generator' for consistency sake across documentation?